### PR TITLE
SW-4141 component improvements

### PR DESCRIPTION
**Pull request by JFSolution**
- imporve usage of AF3
- add logging for resetting usage data
- add glitter felt
- fix 1006 malfunction not showing

### CODE REVIEW

File: octoprint_mrbeam/analytics/usage_handler.py [hunk 1]
```diff
@@ -412,6 +412,7 @@ def reset_prefilter_usage(self, serial):
         self._set_job_time([self.AIRFILTER_KEY, serial, self.PREFILTER_KEY], 0)
         self.start_time_prefilter = -1
         self._write_usage_data()
+        self._logger.info("Reset prefilter usage data")
```
Comment (khaledsherkawi) [inline octoprint_mrbeam/analytics/usage_handler.py]: Please add the percentage value to all the logs

**Review by khaledsherkawi — CHANGES_REQUESTED** [review]

File: octoprint_mrbeam/analytics/usage_handler.py (hunk 1 above)
Comment (JFSolution) [reply octoprint_mrbeam/analytics/usage_handler.py]: done

File: octoprint_mrbeam/analytics/usage_handler.py [hunk 2]
```diff
@@ -920,7 +932,10 @@ def _calculate_af2_filter_usage(self, filter_stage):
 
         # calculate the percentage
         time_percentage = self._get_percentage_from_time(
-            stage_usage_time, self._airfilter.get_lifespans(filter_stage)[0]
+            stage_usage_time,
+            self._airfilter.get_lifespans(filter_stage)[0]
+            if self._airfilter.get_lifespans(filter_stage)
+            else 0.1,
```
Comment (khaledsherkawi) [inline octoprint_mrbeam/analytics/usage_handler.py]: This is not very readable. Maybe define it in a variable beforehand and explain in a comment why 0.1

**Review by khaledsherkawi — CHANGES_REQUESTED** [review]

File: octoprint_mrbeam/analytics/usage_handler.py (hunk 2 above)
Comment (JFSolution) [reply octoprint_mrbeam/analytics/usage_handler.py]: done

**Review by khaledsherkawi — APPROVED** [review]

**Pull request merged**